app/src/main/java/com/kickstarter/libs/KSRendererBuilder.java

```diff
@@ -51,7 +51,7 @@ public KSRendererBuilder(final @NonNull Context context, final @NonNull String v
   @Override
   public void buildRenderers(final @NonNull KSVideoPlayer player) {
     final Allocator allocator = new DefaultAllocator(BUFFER_SEGMENT_SIZE);
-    final DataSource dataSource = new DefaultUriDataSource(context, videoLink);
+    final DataSource dataSource = new DefaultUriDataSource(context, null, videoLink, true);
```

Comment:
so this is the flag that allows redirects?

def makes you miss swift's named arguments when reading this code......